pkg/framework/plugins/defaultevictor/defaultevictor_test.go

```diff
@@ -330,11 +331,13 @@ func TestDefaultEvictorFilter(t *testing.T) {
 					pod.Status.Phase = v1.PodFailed
 				}),
 			},
-		}, {
+		},
```

Comment:
Nit: Keeping the line with `}, {` make the diff smaller. Also, there are many convention used across unit tests. Both `}, {` and `},>NL>{` are acceptable as long as they are not mixed within the same unit test.